src/solver/navier_stokes/source.jl

```diff
@@ -362,16 +373,16 @@ function (obj::SRCPolynomial)(
               t)
   Tdim = 2
   sigma = 0.01
-  gamma = params.gamma
-  gamma_1 = params.gamma_1
-  aoa = params.aoa
+  gamma = params.euler_params.gamma
+  gamma_1 = params.euler_params.gamma_1
+  aoa = params.euler_params.aoa
   q = zeros(typeof(src[1]), Tdim+2)
   q_x = zeros(typeof(src[1]), Tdim, Tdim+2)
   q_xx = zeros(typeof(src[1]), Tdim, Tdim, Tdim+2)
   qRef = zeros(typeof(src[1]), Tdim+2)
   qRef[1] = 1.0
-  qRef[2] = params.Ma*cos(aoa)
-  qRef[3] = params.Ma*sin(aoa)
+  qRef[2] = params.euler_params.Ma*cos(aoa)
```

Comment:
We should use params.uInf here, or I remember there is some function like `getFreeStream`?